picasso/src/main/java/com/squareup/picasso3/FetchAction.java

```diff
@@ -19,6 +19,7 @@
 
 class FetchAction extends Action<Object> {
 
+  @SuppressWarnings("HidingField")
```

Comment:
what's this do?

Reply:
super.target is a different thing.

Reply:
I renamed this and the other field to resolve the hidden/scoping issue.